beerus_cli/src/model.rs

```diff
@@ -239,6 +241,26 @@ pub enum StarkNetSubCommands {
         block_id: String,
     },
     QuerySyncing {},
+    AddDeclareTransaction {
+        /// Max fee
+        #[arg(short, long, value_name = "MAX_FEE")]
+        max_fee: String,
+        /// Max fee
+        #[arg(short, long, value_name = "VERSION")]
+        version: String,
+        /// The signature
+        #[arg(short, long, value_name = "SIGNATURE", value_delimiter = ',')]
+        signature: Vec<String>,
+        /// The nonce
+        #[arg(short, long, value_name = "NONCE")]
+        nonce: String,
+        /// The contract address
```

Comment:
```suggestion
        /// The contract class 
```
Which is the full contract right ?

Reply:
Yes! Correct, hehe thx for fixing the comments!